akka-http/src/main/scala/akka/http/javadsl/unmarshalling/sse/EventStreamUnmarshalling.scala

```diff
@@ -21,8 +22,31 @@ object EventStreamUnmarshalling {
   /**
    * Lets a `HttpEntity` with a `text/event-stream` media type be unmarshalled to a source of `ServerSentEvent`s.
    */
+  @deprecated("Use fromEventStreamWithTimeout", "10.0.10")
```

Comment:
AFAIR we also need to `@Deprecated` for Java people to know, since this is javadsl